bluesky/callbacks/broker.py

```diff
@@ -15,21 +16,28 @@ class LiveImage(CallbackBase):
     field : string
         name of data field in an Event
     """
-    def __init__(self, field, *, fs=None):
+
+    def __init__(self, field, *, fs=None, cmap=None, norm=None,
+                 limit_func=None, auto_redraw=True, interpolation=None):
         from xray_vision.backend.mpl.cross_section_2d import CrossSection
         import matplotlib.pyplot as plt
         super().__init__()
         self.field = field
         fig = plt.figure()
-        self.cs = CrossSection(fig)
+        self.cs = CrossSection(fig, cmap, norm,
+                 limit_func, auto_redraw, interpolation)
         self.cs._fig.show()
         if fs is None:
             import filestore.api as fs
         self.fs = fs
 
     def event(self, doc):
-        uid = doc['data'][self.field]
-        data = self.fs.retrieve(uid)
+        if 'filled' not in doc.keys() or \
```

Comment:
Nik-pick this can be simplified to `if doc.get('filled', {}).get(self.field)` because `None` is false-y.